### DISCUSSION TITLE
Technical Meeting Topics Collection

### DISCUSSION THREAD

**Discussion by Cathy-lulu** [RT-Thread/rt-thread#7980] [Announcements]
![image](https://github.com/RT-Thread/rt-thread/assets/58410639/a38347d8-e538-4f8e-9da6-0a217700c005)

Dear Community Developers,

We’re thrilled to let you know that our Monthly Technical Meeting for community developers is coming up! This is the perfect chance for you to stay in the loop about the latest happenings in the RT-Thread Project and to share your own technical contributions with the community.

To make sure we have a jam-packed agenda, we’re inviting you to submit topics that you’d like to present and share with everyone. This is an awesome opportunity to show off your work and share your expertise with others.

To submit your topic, just comment below with the following details:

- Your name and contact info
- The title of your topic
- A short summary of what you’ll be talking about
- Any extra materials or resources you’d like to share

We can’t wait to see what you’ve got in store for us and to hear all about the amazing work you’ve been doing for the RT-Thread.

**Comment by BernardXiong** [comment]
亲爱的社区开发者，

我们很高兴地通知您，我们面向社区开发者的每月技术会议即将召开！这是您了解 RT-Thread 项目最新动态并与社区分享您自己的技术贡献的绝佳机会。

为了确保我们的议程排得满满的，我们邀请您提交您想要展示并与大家分享的主题。这是一个展示您的工作并与他人分享您的专业知识的绝佳机会。

要提交您的主题，只需在下面评论并提供以下详细信息：

* 您的姓名和联系信息
* 您的主题的标题
* 您将要讨论的内容的简短摘要
* 您想分享的任何额外材料或资源

我们迫不及待地想看看您为我们准备了什么，并想听听您为 RT-Thread 所做的出色工作。
